zephyr/lib/cpu.c

```diff
@@ -120,7 +120,12 @@ static void resume_dais(void)
 		cd = module_get_private_data(mod);
 		dd = cd->dd[0];
 		if (dai_probe(dd->dai->dev) < 0) {
-			tr_err(&zephyr_tr, "DAI resume failed, type %d index %d",
+			tr_err(&zephyr_tr, "DAI resume failed on probe, type %d index %d",
+			       dd->dai->type, dd->dai->index);
```

Comment:
in fact wondering if trying to continue after an error here makes sense? Shouldn't we skip this DAI (using `continune`) if re-probing fails? Same for the `dai_set_config()` failure below?

Reply:
If re-probing fails, setting the config makes no sense and will probably fail itself as well.